cmd/argocd/commands/app.go

```diff
@@ -732,14 +732,17 @@ func addAppFlags(command *cobra.Command, opts *appOptions) {
 // NewApplicationUnsetCommand returns a new instance of an `argocd app unset` command
 func NewApplicationUnsetCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 	var (
-		parameters  []string
-		valuesFiles []string
+		parameters      []string
+		valuesFiles     []string
+		nameSuffix      string
+		namePrefix      string
+		kustomizeImages []string
 	)
 	var command = &cobra.Command{
-		Use:   "unset APPNAME -p COMPONENT=PARAM",
```

Comment:
Can you please add usage example. Similar to https://github.com/argoproj/argo-cd/blob/697e692c0caa832f192ff6734043d8c1f9c6dfdd/cmd/argocd/commands/app.go#L1373-L1377

Reply:
:+1: Code is checked in